RA Scripts/Legend of Zelda, The The Minish Cap.rascript

```diff
@@ -600,22 +600,23 @@ function MusicId() => byte(0x029EF4)
 
 function CreditsLoaded() => MusicId() == 0x09 && Delta(MusicId()) == 0x00
 
-function ScreenIdByte() => byte(0x000BF4)
-function ScreenIdWord() => word(0x000BF4)
+function ScreenId() => word_be(0x000BF4)
+
+function TookDamage() => byte(0x00AAEA) < Delta(byte(0x00AAEA))
```

Comment:
This should be renamed to `JustTookDamage` to emphasize at a glance that there's no hit target and that it's only ever true in the immediate past.